README.md

```diff
@@ -1,20 +1,47 @@
-The PHP Interpreter
-===================
+# The PHP Interpreter
 
-This is the github mirror of the official PHP repository located at
-https://git.php.net.
+[![PHP](http://php.net/images/logos/new-php-logo.png)](https://php.net)
 
 [![Build Status](https://secure.travis-ci.org/php/php-src.svg?branch=master)](http://travis-ci.org/php/php-src)
 [![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy?svg=true)](https://ci.appveyor.com/project/php/php-src)
 
-Pull Requests
-=============
-PHP accepts pull requests via github. Discussions are done on github, but
-depending on the topic can also be relayed to the official PHP developer
-mailing list internals@lists.php.net.
+PHP is a popular general-purpose scripting language that is especially suited to
+web development.
 
-New features require an RFC and must be accepted by the developers.
-See https://wiki.php.net/rfc and https://wiki.php.net/rfc/voting for more
+## Documentation
+
+The PHP manual is available online at https://php.net/docs.php.
+
+## Installation
+
+For installation of PHP, please refer to the online documentation available at
+https://php.net/install.
+
+### Building PHP from source
+
+To build PHP from source on *nix systems:
+
+```
+./buildconf
+./configure
+make
+make test
+sudo make install
```

Comment:
What this step is about?

Reply:
This moves build files to their locations according to the defined folders or a GNU directory structure locations... /usr/local/bin /usr/local/share etc.